custom/icds_reports/mpr_sqldata.py

```diff
@@ -431,6 +433,34 @@ def rows(self):
         return []
 
 
+class MPRUsingSaltBeta(ICDSMixin, MPRData):
+
+    slug = 'using_salt'
+    title = "5. Number of AWCs using Iodized Salt"
+
+    @property
+    def rows(self):
+        if self.config['location_id']:
+            filters = get_location_filter(self.config['location_id'], self.config['domain'])
+            if filters.get('aggregation_level') > 1:
+                filters['aggregation_level'] -= 1
+
+            filters['month'] = date(self.config['year'], self.config['month'], 1)
+
+            awc_data = AggAwc.objects.filter(**filters).values(
+                'use_salt',
+                'num_launched_awcs').order_by('month').first()
+
+            use_salt = awc_data.get('use_salt', 0)
+            percent = "%.2f" % ((use_salt or 0) * 100 / float(awc_data.get('num_launched_awcs', 0) or 1))
```

Comment:
is `awc_number` in the previous definition only launched?

Reply:
hey, no they were the total number of awcs in that area(launched+unlaunched).
While going through the MPR calculations. It was decided to just include launched AWC as the denominator of this